reframe/core/pipeline.py

```diff
@@ -703,11 +703,11 @@ def pipeline_hooks(cls):
     #: :type: boolean : :default: :class:`True`
     build_locally = fields.TypedField('build_locally', bool)
 
-    def __new__(cls, *args, **kwargs):
+    def __new__(cls, *args, _rfm_use_params=False, **kwargs):
```

Comment:
Are you sure that this can ever be passed as true? Since it's consumed by the metaclass.

Reply:
Ignore this, I haven't seen closely the metaclass' `__call__()` implementation.